components/eamxx/mkdocs.yml

```diff
@@ -47,7 +46,6 @@ theme:
   features:
     - navigation.indices
     - navigation.instant
-    - navigation.sections
```

Comment:
Removing this allows to collapse the user/developer/technical sections in the home page of eamxx docs, which may be useful if we start adding several pages.